### ISSUE TITLE
graphql: update client definitions with gas price config

### ISSUE THREAD

**Issue by renaynay** [ethereum/hive#407]
Update client definitions with gas price config to fix the inconsistency between Besu and Geth below: 

```
Name
07_eth_gasPrice (go-ethereum)

Description
Test case source: https://github.com/ethereum/hive/blob/master/simulators/ethereum/graphql/testcases/07_eth_gasPrice.json

Details

Test failed.
HTTP response code: 200 OK
expected: {
  "data": {
    "gasPrice": "0x10"
  }
}
got: {
  "data": {
    "gasPrice": "0x1"
  }
}
```

**Comment by fjl** [comment]
I have fixed this in #418. All clients now use 16 gwei gas price. Not sure if this requires any changes in graphql suite.

**Comment by renaynay** [comment]
Oh yes this issue can be closed.
